File: 2012/November/26-11-2012/spec/FizzBuzzSpec.js
describe("FizzBuzz", function() {
  var fizzBuzz;

  beforeEach(function(){
    fizzBuzz = new FizzBuzz();
  })
  it("should return fizz given 3", function() {
    expect(fizzBuzz.calculateFizz(3)).toEqual("fizz");
  });

  it("should return buzz given 5", function() {
    expect(fizzBuzz.calculateBuzz(5)).toEqual("buzz");
  });

  it("should return fizzbuzz given 15", function() {
    expect(fizzBuzz.calculateFizzBuzz(15)).toEqual("fizzbuzz");
  });
})

File: 2012/November/27-11-2012/fizzbuzz_spec.rb
require_relative 'fizzbuzz'

describe 'FizzBuzz' do
  let(:fizzbuzz) {FizzBuzz.new}
  it "should return the number by default" do
    fizzbuzz.calculate(1).should == "1"
  end

  it "should return 'fizz' when number is dividable by 3" do
    fizzbuzz.calculate(3).should == "fizz"
  end

  it "should return 'buzz' when number is dividable by 5" do
    fizzbuzz.calculate(5).should == "buzz"
  end

  it "should return 'fizzbuzz' when number is dividable by 15" do
    pending "add tests"
  end
end